apps/coscrad-frontend/src/utils/string-processor/shorten-string.ts

```diff
@@ -0,0 +1,6 @@
+export const shortenString = (text: string, maxNumberOfChars: number): string => {
```

Comment:
No need to define a const `shortenedString`. Just use an implicit return here ( no need for `{...}`).